CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [v8.1.0] - 2020-07-23
+### Added
+- Add optional (turned on by `add_suffix` parameter) random suffix to all resources that will prevent more instances of this module to be provisioned
```

Comment:
@sandratatarevicova grammar nazi opinion required :-) is it postfix or suffix?

Reply:
I think suffix is used more frequently in this context. I would change the variable name to `add_random_suffix` so that it is more self-explanatory.

Reply:
I agree, changed :)